lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -218,7 +218,7 @@ export default class DocumentComposer {
 
     const publicKeyIdSet: Set<string> = new Set();
     for (const publicKey of publicKeys) {
-      const allowedProperties = new Set(['id', 'type', 'purposes', 'publicKeyJwk']);
+      const allowedProperties = new Set(['id', 'type', 'purposes', 'publicKeyJwk', 'controller']);
```

Comment:
The linked spec says the controller must be a DID and optionally not exceed some maximum length. Need to add such checks and tests accordingly.